src/PatientChecking/PatientChecking/Controllers/AppointmentController.cs

```diff
@@ -1,16 +1,57 @@
 ﻿using Microsoft.AspNetCore.Mvc;
+using PatientChecking.Services.Repository;
+using PatientChecking.Services.ServiceModels;
+using PatientChecking.Services.ServiceModels.Enum;
+using PatientChecking.Views.ViewModels;
 using System;
 using System.Collections.Generic;
+using System.Dynamic;
 using System.Linq;
 using System.Threading.Tasks;
 
 namespace PatientChecking.Controllers
 {
     public class AppointmentController : BaseController
     {
-        public IActionResult Index()
+        private readonly IAppointmentService _appointmentService;
+
+        public AppointmentController(IAppointmentService appointmentService)
+        {
+            _appointmentService = appointmentService;
+        }
+
+        public async Task<IActionResult> Index(int Option = (int)AppointmentSortSelection.ID, int PageSize = 10, int PageIndex = 1)
```

Comment:
The parameter in method should be camelCase. Please check for other places.

Reply:
camelCase has applied to the parameter in method